action.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
+import json
+import logging
+import math
+import os
+import subprocess
+import tempfile
+import time
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+from zipfile import ZipFile
+
+import boto3
+
+
+@dataclass
+class ContainerConfig:
+    name: str
+    dockerfile_path: str
+    image_base_url: str
+    host_port: int
+    container_port: int
+    memory: Optional[int] = 1024
+    links: Optional[list[str]] = None
+
+    def image_url(self, version):
```

Comment:
Kann das eine property sein?

Reply:
Ich glaube, dass das `version` sich je nach Deployment ändert. Die ContainerConfig bleibt aber gleich. Wahrscheinlich wäre dann folgendes eine Idee:
```suggestion
    def get_image_url(self, version):
```

Reply:
Genau, Version ist in dem Fall immer der `commit`, welcher in die Action reingegeben wird.
Ich werde die Funktion umbennen, wie @mm-matthias vorgeschlagen hat.